src/components/CreateLink.js

```diff
@@ -15,57 +14,43 @@ const POST_MUTATION = gql`
   }
 `
 
-class CreateLink extends Component {
-  state = {
-    description: '',
-    url: '',
-  }
+const CreateLink = ({ history }) => {
+  const [description, setDescription] = React.useState("");
+  const [url, setUrl] = React.useState('');
+  const [state, executeMutation] = useMutation(POST_MUTATION);
```

Comment:
They pass their variables straight into this, I don't know if we want to do this too